core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -221,7 +221,7 @@ public void scanAndApply( final ManipulationSession session )
 
                 WildcardMap<ProjectVersionRef> map = (session.getState( RelocationState.class) == null ? new WildcardMap<>() : session.getState( RelocationState.class ).getDependencyRelocations());
                 String report = ProjectComparator.compareProjects( session, jsonReport, map , originalProjects, currentProjects );
-                logger.info( report );
+                logger.info( "{}{}", System.lineSeparator(), report );
```

Comment:
This adds a newline to the log so that the first line of the output is not misaligned.

Reply:
Nice!